### ISSUE TITLE
Profile Icon Link Inaccurate Alt Text

### ISSUE THREAD

**Issue by Centerville1** [gordon-cs/gordon-360-ui#2134]
The profile icon link on the header currently says "button: more."  on a screen reader.  This is likely left over from when it was a nav menu, but it should be updated to something more descriptive.
